cql3/expr/expression.cc

```diff
@@ -428,12 +428,8 @@ namespace {
 bool_or_null like(const expression& lhs, const expression& rhs, const evaluation_inputs& inputs) {
     data_type lhs_type = type_of(lhs)->underlying_type();
     if (!lhs_type->is_string()) {
-        expression::printer lhs_printer {
-            .expr_to_print = lhs,
-            .debug_mode = false
-        };
         throw exceptions::invalid_request_exception(
-                format("LIKE is allowed only on string types, which {} is not", lhs_printer));
+                format("LIKE is allowed only on string types, which {:user} is not", lhs));
```

Comment:
Can `{:user}` be made the default so that just `{}` will just do what `{:user}` does? I assume that 1. most printouts will be to the user and want this format and 2. even if we mix up the call, it's better to get the user printouts right, so `{}` will do the right thing for user printouts.

Reply:
Oh! I see now that you've made `{:debug}` the default. I think this is a mistake for the reasons I explained above.